env_manager/backends/conda_like_interface.py

```diff
@@ -105,9 +125,14 @@ def install_packages(
             channels = ["-c"] + channels
             command += channels
         try:
-            result = subprocess.run(
-                command, stderr=subprocess.PIPE, check=True, text=True
-            )
+            if capture_output:
+                result = subprocess.run(
+                    command, capture_output=capture_output, check=True, text=True
```

Comment:
```suggestion
                    command, capture_output=True, check=True, text=True
```

Minor suggestion but I think it'd make the code easier to read.